tests/src/JIT/SIMD/Vector3Interop.cs

```diff
@@ -71,19 +118,59 @@ public static bool test()
             }
         }
         
+        // Argument passing test for Unix.
+        // Few arguments are passed onto stack.
+        {
+            Vector3 v1 = new Vector3(1, 2, 3);
+            Vector3 v2 = new Vector3(10, 20, 30);
+            float f0 = 100, f1 = 101, f2 = 102, f3 = 103, f4 = 104, f5 = 105, f6 = 106, f7 = 107, f8 = 108;
+            
+            float sum = nativeCall_PInvoke_Vector3Arg_Unix(
+                v1, // register
+                f0, f1, f2, f3, f4, f5, // register
+                f6, v2,  // stack
+                f7, f8); // stack
```

Comment:
We have modified signature of nativeCall_PInvoke_Vector3Arg_Unix() but i see this method call is not passing additional args? Won't this lead to a compilation error or am i missing something?